data/json/items/tool_armor.json

```diff
@@ -2306,5 +2306,70 @@
         "bashing" : 12,
         "coverage" : 20,
         "material_thickness" : 4    
+    },
+    {
+        "id": "powered_earmuffs",
+        "type": "TOOL_ARMOR",
+        "category" : "armor",
+        "symbol": "[",
+        "color": "blue",
+        "name": "shooter's earmuffs",
```

Comment:
It's better to define ```"plural_name"``` elsewise game will use standard pluralization rule and you will get a bunch of  shooter's earmuffss (with double 's' at the end).